tests/test_ssl_self_signed.py

```diff
@@ -35,6 +35,7 @@ def test_001_init_primary():
     node1.run()
     assert node1.wait_until_state(target_state="single")
 
+    node1.wait_until_pg_is_running()
```

Comment:
I was suprised this is needed (or the one below for that matter). Because if a node is "single" (or "secondary"), it should already be running right? I guess it's because we test target_state instead of current_state.

Reply:
We test the reported state as seen on the monitor, so that should mean we're good and Postgres is running. Still, it seems like having that reliable in the test suite without the extra explicit wait was not possible for me.